echopype/echodata/zarr_combine.py

```diff
@@ -627,6 +628,7 @@ def write_to_file(
         """
 
         with Lock(lock_name):
+            # Write to region is not consolidated by default
```

Comment:
I am not seeing this in the docs anywhere. Perhaps you found this by looking at the actual code? Either way, I think it would be best to be explicit here. Could you please add `consolidated=False` in `ds_in.to_zarr`?